bcfs/src/bcfs.rs

```diff
@@ -130,6 +121,21 @@ impl<A: Address, M: AccountMeta> BCFS<A, M> {
         Ok(fd)
     }
 
+    pub fn tempfile(
```

Comment:
A comment explaining why one would use this method would be useful for readers.

My understanding is that it's used for writing the output data for a cross-service call. If that is the case, why do we create a tempfile for each cross-service call instead of having one designated file for all cross-service calls?

Reply:
> A comment explaining why one would use this method would be useful for readers.

Really? It's exactly [POSIX `tempfile`](https://linux.die.net/man/3/tmpfile)

> why do we create a tempfile for each cross-service call 

We don't. After the output is read, the [file is dropped](https://github.com/oasislabs/oasis-rs/blob/master/oasis-std/src/backend/wasi.rs#L95-L97). [The `Drop` impl](https://github.com/rust-lang/rust/blob/master/src/libstd/sys/wasi/fd.rs#L347) closes the file which frees the memory.